tests/meta/upload/conftest.py

```diff
@@ -2,18 +2,31 @@
 
 import json
 
+from pathlib import Path
+from datetime import datetime
+
 import pytest
 
 from cg.apps.coverage.api import ChanjoAPI
 from cg.meta.upload.coverage import UploadCoverageApi
 from cg.meta.upload.mutacc import UploadToMutaccAPI
 from cg.meta.upload.observations import UploadObservationsAPI
 from cg.meta.upload.scoutapi import UploadScoutAPI
+from cg.meta.upload.gt import UploadGenotypesAPI
+from cg.store import models
+from cg.store import Store
+
+
+# Mocks
 
 
 class MockAnalysis:
     """Mock an analysis object"""
 
+    def __init__(self, started_at: datetime = None, completed_at=None):
+        self.started_at = started_at or datetime.now()
```

Comment:
I know this might be out of scope of this PR, but could you help clean up this mock? Both the name and the docstring do not describe what this is for. Its a mock of AnalysisAPI (which I should have renamed MIPAnalysisAPI :facepalm: ) But also there are multiple things in cg which could be considered an "analysis object"

At the same time, can you elaborate why those init values are there? Started at and completed at are properties of an Analysis in ClinicalDB, and not AnalysisAPI. This mock seems to be both...

Reply:
Have I done that?

Reply:
Reverted this change